posthog/async_migrations/utils.py

```diff
@@ -31,14 +32,22 @@ def execute_op_postgres(sql: str, query_id: str):
         cursor.execute(f"/* {query_id} */ " + sql)
 
 
-def process_error(migration_instance: AsyncMigration, error: Optional[str], rollback: bool = True):
+def process_error(migration_instance: AsyncMigration, error: str, rollback: bool = True, alert: bool = False):
```

Comment:
I'd really like to keep the `Optional[str]` around or better yet a simple `Optional[Error]` type like you had would be nice.

Reply:
why?